UIViewController-DisplayChild/UIViewController+DisplayChild.swift

```diff
@@ -102,7 +102,17 @@ public extension UIViewController {
 
     func childViewController<T>(in optionalContainerView: UIView? = nil) -> T? {
         let containerView = optionalContainerView ?? view!
-        return childViewController(at: containerView) as? T
+        let result: T?
+        //see also `presentedController(with:)
+        if let controller = childViewController(at: containerView) as? T {
+            result = controller
+        } else if let navigationController = childViewController(at: containerView) as? UINavigationController,
+            let controller = navigationController.viewControllers[0] as? T {
```

Comment:
this will not find child even if it is embedded in UINavigationController. how about filter first of type in viewControllers?